### DISCUSSION TITLE
getting an error when trying to build firmware

### DISCUSSION THREAD

**Discussion by Xyynel** [mriscoc/Ender3V2S1#1355] [General]
i use the custom config app and when i go to build in auto marlin i get a failed error  that says 
c/users/owner/.platformio/packaged/toolchain-gcarmnoeeabi/bin/.../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none/eabi/bin/ld.exe adress 0x80418dc of .pio/build/stm32f401rc_creailty/firmware.elf section '.text' is not within region 'flash'
repeats that 3 times but if i build the 512k it does it just fine am i doing something wrong? please help

**Comment by mriscoc** [comment]
STM32F401RCT6 chips doesn't have enough program memory for many features.
